qa/tasks/cephfs/test_volumes.py

```diff
@@ -95,6 +95,27 @@ def tearDown(self):
             self._delete_test_volume()
         super(TestVolumes, self).tearDown()
 
+    def test_volume_rm(self):
+        try:
+            self._fs_cmd("volume", "rm", self.volname)
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.EPERM:
+                raise RuntimeError("expected the 'fs volume rm' command to fail with EPERM, " \
```

Comment:
You don't need a `\` to break a long line when you're enclosing the string within brackets and using quotes?

Reply:
Right, I will update it.